### PR TITLE
Add extending block settings docs

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q | A
| --- | ---
| Fixed tickets | fixes -
| Related PRs | sulu/sulu#prnum
| License | MIT

#### What's in this PR?

Add extending block settings docs.

#### Why?

Sometimes you want to add custom properties to the block settings.

### CODE REVIEW

**Comment by TheCadien** [comment]
Thanks for fast adding it !

File: reference/content-types/block.rst [hunk 1]
```diff
@@ -119,3 +119,62 @@ A reusable way for rendering blocks is having a separate template file per type:
...
+                                </param>
+                            </param>
+                        </params>
+                    </property>
+                </properties>
+            </section>
+        </properties>
+    </form>
+
+The settings can be accessed in twig over the ``settings`` variable:
```
Comment (nnatter) [inline reference/content-types/block.rst:172]:
```suggestion
The value of your field can be accessed in twig over the ``settings`` variable:
```

**Review by nnatter — COMMENTED** [review]
Would like to resolve the comments on https://github.com/sulu/sulu-demo/pull/86/files before merging this 🙂

**Comment by alexander-schranz** [comment]
@nnatter updated

File: reference/content-types/block.rst [hunk 2]
```diff
@@ -119,3 +119,62 @@ A reusable way for rendering blocks is having a separate template file per type:
...
+                        <meta>
+                            <title lang="en">Block Theme</title>
+                            <title lang="de">Block Theme</title>
+                        </meta>
+
+                        <params>
+                            <param name="default_value" value="default"/>
+
+                            <param name="values" type="collection">
+                                <param name="default">
```
Comment (nnatter) [inline reference/content-types/block.rst:151]:
```suggestion
                            <param name="default_value" value=""/>

                            <param name="values" type="collection">
                                <param name="">
```

**Review by nnatter — APPROVED** [review]

**Pull request merged**